.github/workflows/build.yml

```diff
@@ -309,11 +309,25 @@ jobs:
           /usr/bin/python3
       - name: Install Packer plugins
         run: packer init .
+      - id: get-third-party-bucket-name
+        name: Get the third-party bucket name from SSM Parameter Store
+        run: |
+          echo name=$(aws ssm get-parameter \
+            --name /third_party_bucket_name \
+            --output text \
+            --query Parameter.Value \
+            --with-decryption) >> $GITHUB_OUTPUT
       - name: Create machine image
+        env:
+          # Since we are using the default value of 15 seconds for
+          # AWS_POLL_DELAY_SECONDS, this corresponds to a timeout of
+          # an hour and a half.  We need such a long timeout because
+          # it can take almost an hour to create this AMI.
+          AWS_MAX_ATTEMPTS: 360
```

Comment:
Why is this here but not in the `release.yml` or `prerelease.yml` workflows?  I suspect it isn't needed.

Reply:
It is currently in both the release and prerelease workflows - please see the last line of my commit message in https://github.com/cisagov/openvpn-packer/pull/120/commits/e74370929fee9a0d4550c1be4aa24718b91832d6.

The alternative is to remove it from all of the workflows, since the build workflow has been running successfully without it, maybe the same would be true for the prerelease and release workflows as well.